e2e/cypress/tests/integration/performance/team_switch_spec.js

```diff
@@ -33,7 +33,7 @@ describe('Channel switch performance test', () => {
 
     it('measures switching between two teams from LHS', () => {
         // # Invoke window object
-        measurePerformance('teamLoad', 5000, () => {
+        measurePerformance('teamLoad', 900, () => {
```

Comment:
Do you think we could drop these even lower? I'm wondering what sort of variance you were seeing with them so far